TypeScript error: "Property 'raws' does not exist on type 'Combinator'."

## Summary

Hi. I encounter a TypeScript error when using a `Combinator`. It seems that the `raws` property on the `Combinator` type is missing.

Here is a reproduction: ⬇️ 

## Reproduction

`package.json`:

```json
{
  "dependencies": {
    "postcss-selector-parser": "6.0.8",
    "typescript": "4.5.4"
  }
}
```

`a.js`:

```js
const parser = require('postcss-selector-parser');

parser((selectors) => {
	selectors.walkCombinators((comb) => {
		console.log(comb.raws);
	});
}).processSync('a + b');
```

Run:

```console
$ npm install
...

$ node a.js
{ spaces: { before: undefined } }

$ ./node_modules/.bin/tsc --checkJs --noEmit a.js
a.js:5:20 - error TS2339: Property 'raws' does not exist on type 'Combinator'.

5   console.log(comb.raws);
                     ~~~~


Found 1 error.
```

## Environment

- `postcss-selector-parser` 6.0.8
- TypeScript 4.5.4
- Node.js 17.3.1
- npm 8.3.0
- macOS Monterey 12.1

--- a/postcss-selector-parser.d.ts
+++ b/postcss-selector-parser.d.ts
@@ -339,8 +339,16 @@ declare namespace parser {
     function selector(opts: ContainerOptions): Selector;
     function isSelector(node: any): node is Selector;
 
+    interface CombinatorRaws {
+        value?: string;
+        spaces?: {
+            before?: string;
+            after?: string;
+        };
+    }
     interface Combinator extends Base {
-        type: "combinator"
+        type: "combinator";
+        raws?: CombinatorRaws;
     }
     function combinator(opts: NodeOptions): Combinator;
     function isCombinator(node: any): node is Combinator;

